### ISSUE TITLE
Mongoid 5 support? 

### ISSUE THREAD

**Issue by steveo7** [haihappen/mongoid-versioning#6]
Any chance this will be updated to support Mongoid 5?  It's been working great, but it's dependent on Mongoid 4.

**Comment by haihappen** [comment]
Can you create a fork and test Mongoid 5 support?

**Comment by steveo7** [comment]
I did a cursory test of my app and everything works.  The only thing I had to change in mongoid-versioning is the version of mongoid.  There a few changes I had to make for Mongoid that are documented on the mongodb site, https://docs.mongodb.org/ecosystem/tutorial/ruby-mongoid-tutorial/#upgrading-to-5-0-0

Of course this is just basic testing, but so far so good.

**Comment by haihappen** [comment]
There is still a problem which I couldn't figure out yet:

```
Failures:

  1) Mongoid::Versioning#version when not using the default database when the document is persisted more than once persists the versions to specified database
     Failure/Error: expect(WikiPage.with(database: "mongoid_test_alt").find_by(:title => title).version).to eq(4)
       
       expected: 4
            got: 1
       
       (compared using ==)
     # ./spec/mongoid/versioning_spec.rb:232:in `block (5 levels) in <top (required)>'

Finished in 0.69231 seconds (files took 0.43372 seconds to load)
52 examples, 1 failure

Failed examples:

rspec ./spec/mongoid/versioning_spec.rb:231 # Mongoid::Versioning#version when not using the default database when the document is persisted more than once persists the versions to specified database
```

**Comment by steveo7** [comment]
You might to look at honest's fork.  https://github.com/honest/mongoid-versioning/tree/mongoid-5-compatible

**Comment by haihappen** [comment]
Unfortunately @honest's fork has the same problems 😕

**Comment by haihappen** [comment]
Weird! After dropping all my local dbs and rerunning the tests they suddenly worked.

**Comment by haihappen** [comment]
Never mind, still having the errors on my local machine and on Travis.

**Comment by haihappen** [comment]
Fixed, and 1.2.0 with Mongoid 5 support released 🙂

**Comment by steveo7** [comment]
WAHOO!!
